gateleen-monitoring/src/main/java/org/swisspush/gateleen/monitoring/RedisMonitor.java

```diff
@@ -115,8 +115,12 @@ public String apply(@Nullable String input) {
                     String[] pairs = val.split(",");
                     for (String pair : pairs) {
                         String[] tokens = pair.split("=");
-                        value = Long.parseLong(tokens[1]);
-                        publisher.publishMetric("keyspace." + key + "." + tokens[0], value);
+                        if(tokens.length == 2) {
+                            value = Long.parseLong(tokens[1]);
+                            publisher.publishMetric("keyspace." + key + "." + tokens[0], value);
+                        } else {
+                            log.warn("Invalid keyspace property. Will be ignored");
```

Comment:
Any reason to not log any details? For example I think contents of "*pair*" probably would be very interesting for bug-hunting. Like:
```java
log.warn("Invalid keyspace property. Will be ignored: {}", pair)
```